<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>üìö Library Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
QWidget {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #fffbe7, stop:1 #e0c097);
    font-family: 'Georgia', 'Times New Roman', serif;
}
    </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0">
    <item>
     <widget class="QLabel" name="labelHeader">
      <property name="styleSheet">
       <string notr="true">
font-size: 28px;
font-weight: bold;
color: #4e342e;
margin-top: 18px;
margin-bottom: 12px;
font-family: 'Georgia', 'Times New Roman', serif;
       </string>
      </property>
      <property name="text">
       <string>üìö Library Management System</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelUserInfo">
      <property name="text">
       <string>Logged in as: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">
font-size: 16px;
color: #6d4c41;
font-family: 'Georgia', 'Times New Roman', serif;
margin-bottom: 8px;
       </string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLineEdit" name="lineEditSearch">
        <property name="styleSheet">
         <string notr="true">
QLineEdit {
    border: 2px solid #a1887f;
    border-radius: 14px;
    padding: 6px 14px;
    font-size: 16px;
    background: #fffbe7;
    margin-bottom: 8px;
    font-family: 'Georgia', 'Times New Roman', serif;
}
         </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>üîé Search resources by title or author‚Ä¶</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxCategory">
        <property name="styleSheet">
         <string notr="true">
QComboBox {
    border: 2px solid #a1887f;
    border-radius: 14px;
    padding: 6px 14px;
    font-size: 16px;
    background: #fffbe7;
    margin-bottom: 8px;
    font-family: 'Georgia', 'Times New Roman', serif;
}
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxStatus">
        <property name="styleSheet">
         <string notr="true">
QComboBox {
    border: 2px solid #a1887f;
    border-radius: 14px;
    padding: 6px 14px;
    font-size: 16px;
    background: #fffbe7;
    margin-bottom: 8px;
    font-family: 'Georgia', 'Times New Roman', serif;
}
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSearch">
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    background-color: #4e342e;
    color: #fffbe7;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    letter-spacing: 1px;
    box-shadow: 0 4px 16px #4e342e33;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #3e2723;
}
         </string>
        </property>
        <property name="text">
         <string>üîç Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetResources">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QTableWidget {
    background: #fffbe7;
    border-radius: 18px;
    font-size: 15px;
    gridline-color: #d7ccc8;
    margin-top: 8px;
    margin-bottom: 8px;
    box-shadow: 0 8px 32px #8d6e6322;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QHeaderView::section {
    background-color: #4e342e;
    color: #fffbe7;
    font-weight: bold;
    font-size: 15px;
    border: none;
    border-radius: 0px;
    padding: 10px;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QTableWidget::item:selected {
    background: #ffe082;
    color: #4e342e;
}
       </string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Author</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Borrower</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reserver</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="actionLayoutVertical">
      <item>
       <layout class="QHBoxLayout" name="actionRow1">
        <item>
         <widget class="QPushButton" name="buttonAdd">
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    background-color: #8d6e63;
    color: #fffbe7;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    box-shadow: 0 4px 16px #8d6e6333;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #6d4c41;
}
           </string>
          </property>
          <property name="text">
           <string>‚ûï Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonEdit">
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    background-color: #ffe082;
    color: #4e342e;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    box-shadow: 0 4px 16px #ffe08233;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #ffd54f;
}
           </string>
          </property>
          <property name="text">
           <string>‚úèÔ∏è Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRemove">
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    background-color: #b71c1c;
    color: #fffbe7;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    box-shadow: 0 4px 16px #b71c1c33;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #7f0000;
}
           </string>
          </property>
          <property name="text">
           <string>üóëÔ∏è Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonBorrow">
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    background-color: #388e3c;
    color: #fffbe7;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    box-shadow: 0 4px 16px #388e3c33;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #1b5e20;
}
           </string>
          </property>
          <property name="text">
           <string>üìñ Borrow</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="actionRow2">
        <item>
         <widget class="QPushButton" name="buttonReserve">
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    background-color: #b71c1c;
    color: #fffbe7;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    box-shadow: 0 4px 16px #b71c1c33;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #7f0000;
}
           </string>
          </property>
          <property name="text">
           <string>üîñ Reserve</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonReturn">
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    background-color: #fbc02d;
    color: #4e342e;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    box-shadow: 0 4px 16px #fbc02d33;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #ff8f00;
}
           </string>
          </property>
          <property name="text">
           <string>üîÑ Return</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRenew">
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    background-color: #00897b;
    color: #fffbe7;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    box-shadow: 0 4px 16px #00897b33;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #004d40;
}
           </string>
          </property>
          <property name="text">
           <string>‚ôªÔ∏è Renew</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonViewHistory">
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    background-color: #6d4c41;
    color: #fffbe7;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    box-shadow: 0 4px 16px #6d4c4133;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #4e342e;
}
           </string>
          </property>
          <property name="text">
           <string>üìú View History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSchedule">
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    background-color: #a1887f;
    color: #4e342e;
    border-radius: 14px;
    padding: 8px 24px;
    font-size: 16px;
    font-weight: bold;
    box-shadow: 0 4px 16px #a1887f33;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #8d6e63;
}
           </string>
          </property>
          <property name="text">
           <string>üïí Schedule &amp; Events</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
